integrated_channels/blackboard/client.py

```diff
@@ -610,14 +619,18 @@ def create_integration_content_for_course(self, bb_course_id, channel_metadata_i
             content_child_id = course_fully_created_response.json().get('id')
         except ClientError as error:
             content_child_id = None
+            error_message = ' and received error response={}'.format(error.message)
+            is_error = True
 
         if not content_child_id:
-            error_handling_response = self.delete_course_from_blackboard(bb_course_id, bb_content_id)
+            error_message = error_message if is_error else ''
```

Comment:
if you decide to refactor as I had recommended, this comment will may be somewhat impacted but do existing tests cover this error handling change? may want to review/revisit if we want a more pointed test to test that we delete (or do not delete) in response to an error using mocks assert_called counts check maybe, but leave it upto you